UM/Qt/Bindings/Theme.py

```diff
@@ -32,36 +35,90 @@ def __init__(self, engine, parent = None):
         self._em_height = int(QFontMetrics(QCoreApplication.instance().font()).ascent())
         self._em_width = self._em_height;
 
-        self._initializeDefaults()
+        Preferences.getInstance().addPreference("general/theme", Application.getInstance().getApplicationName())
+
+        try:
+            theme_path = Resources.getPath(Resources.Themes, Preferences.getInstance().getValue("general/theme"))
+            self.load(theme_path)
+        except FileNotFoundError:
+            pass
```

Comment:
What do you mean with "you might want to move the win32 font workaround into _initializeDefaults."? The system font that is initialized by _initializeDefaults is already the default system font and will be correct on Windows. The workaround when loading fonts is to replace the requested font family with the system font family, since other fonts really do not render well on Windows. Since _initializeDefaults does not load any other fonts, I do not see what I need to change there.